ZenPacks/zenoss/Microsoft/Windows/Device.py

```diff
@@ -121,46 +121,56 @@ def getClusterMachinesList(self):
                     clusterdnsname))
         return _clusterdevices
 
+    def is_iis(self):
+        '''Return True if an IIS server'''
+        # if we have IIS components, then we are IIS server
+        for component in self.getDeviceComponents():
+            if isinstance(component, WinIIS):
+                return True
+        return False
+
+    def is_ntds(self):
+        '''Return True if an NTDS/AD server'''
+        # running the NTDS service indicates that this is an AD server
+        for component in self.getDeviceComponents():
```

Comment:
can you remove this check?  apparently checking for ntds is not the best way as a server can have lightweight ad on it and not be a domain controller.  I added a fix in the os modeler plugin recently that is the best way to detect a dc so we should only look at the self.domain_controller property.